### ISSUE TITLE
AppImage core dump related to OpenGL

### ISSUE THREAD

**Issue by nslogan** [mavlink/qgroundcontrol#7071]
## Expected Behavior

QGC to start up.

## Current Behavior

```
Settings location "/home/logan/.config/QGroundControl.org/QGroundControl.ini" Is writable?: true
Filter rules "*Log.debug=false\n"
QGeoPositionInfoSourceGeoclueMaster(0x292ed80)
Failed to set Geoclue positioning requirements. Geoclue error: org.qtproject.QtDBus.Error.InvalidObjectPath
QGCPositionManager error 2
MAVLinkLogManagerLog: MAVLink logs directory: "/home/logan/Documents/QGroundControl/Logs"
Map Cache in: "/home/logan/.cache/QGCMapCache300" / "qgcMapCache.db"
QGLXContext: Failed to create dummy context
Failed to create OpenGL context for format QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize 24, redBufferSize -1, greenBufferSize -1, blueBufferSize -1, alphaBufferSize -1, stencilBufferSize 8, samples -1, swapBehavior QSurfaceFormat::SwapBehavior(DoubleBuffer), swapInterval 1, colorSpace QSurfaceFormat::ColorSpace(DefaultColorSpace), profile  QSurfaceFormat::OpenGLContextProfile(NoProfile)) 
Aborted (core dumped)
```

## Steps to Reproduce:

Try to launch AppImage on a similarly configured system.

## System Information

```
$ uname -a
Linux pop-os 4.15.0-39-generic #42-Ubuntu SMP Tue Oct 23 15:48:01 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

Pop!OS is [built off of the Ubuntu 18.04 repositories](https://pop.system76.com/docs/difference-between-pop-ubuntu/).

I ran `apt update` yesterday and discovered QGC was broken this morning when I tried to run it for the first time today. I had the same SSL issue as #7063 which I solved by running `apt install  openssl1.0 libssl1.0-dev`.

- QGC Version: 3.4.4
- QGC build: AppImage, latest

I can provide additional system information as needed.

**Comment by DonLakeFlyer** [comment]
You can re-open with more details after looking through the search results for solutions. But unlikely that there is much we can do about it since Qt has certain requirements to run.

**Comment by nslogan** [comment]
I ran this *same* AppImage yesterday prior to running `apt update` and it functioned properly, so I'd be surprised if it's related to my graphics card or driver age.

The update I ran included updates to my graphics driver, going from `nvidia-396` to `nvidia-410`.

I did Google around for a while attempting to solve this problem but haven't been able to track anything down. I don't have the time at the moment to downgrade my NVIDIA driver to 396. However, I had another machine with the same version of Pop!OS that hasn't had the graphics drivers updated and 3.4.4 still runs on it (obviously this isn't a one-to-one test but it's at least a hint). If I can find yet another machine to run QGC on so I can keep functioning today I will try updating the drivers on that Pop!OS machine and see if it breaks QGC as well.
